components/table/style/index.less

```diff
@@ -143,10 +145,9 @@
         }
       }
 
-      &.@{table-prefix-cls}-row:hover {
-        > td {
-          background: @table-row-hover-bg;
-        }
+      &.@{table-prefix-cls}-row:hover > td,
+      > td.@{table-prefix-cls}-cell-row-hover {
```

Comment:
nit: if we need to add `ant-table-cell-row-hover` dynamically, we should use `fromEvent` to listen the event named `mouseover/mouseout`, will do that after styles checked.